### ISSUE TITLE
Know more Links point to google code labs website

### ISSUE THREAD

**Issue by SoumavaBanerjee** [Hack-Club-UEMK/hackclub.github.io#5]
#### THE ISSUE:
- All the **know more** links in the technologies part  points to the same link.

### EXPECTED BEHAVIOUR:
- All the links should point to a single form.

Please fix this.

**Comment by SoumavaBanerjee** [comment]
Just create a google form and put it's link on the a tags.

**Comment by Prasundas99** [comment]
I have a better idea as of now lets have a Repo or a html page  for each know more topics and write something about it, i.e what is Hack club ML is about what does they expects

**Comment by krohan1202** [comment]
Yeah... @Prasundas99 that will be better. However, under the topics, there should also be projects. We can add private repos but at this time, private won't do us good.

**Comment by Prasundas99** [comment]
Just create a github repo and connect its readme files to it @krohan1202

**Comment by SoumavaBanerjee** [comment]
Okay, that will be great

**Comment by krohan1202** [comment]
Assigned new repo to me for docs by @Prasundas99
